server/dba/migrations/33-speciesNotes.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   up: function(queryInterface, Sequelize) {
```

Comment:
Why hasn't this table been deprecated?

Reply:
@lauraGgit created story to move this to JSON as discussed earlier: https://github.com/nciinc/fs-intake-module/issues/104